features/adb-openshift.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Command output from various OpenShift related commands in ADB
+  service-manager should return the correct output from commands affecting OpenShift in ADB
+
+  @ci-only
+  Scenario Outline: Boot and execute commands
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    begin
+      require 'vagrant-libvirt'
+    rescue LoadError
+      # NOOP
+    end
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker, openshift'
+
+      config.servicemanager.openshift_image_tag = 'v1.2.1'
```

Comment:
@hferentschik Just to give more context why I added these three magically lines.
There is a issue with fetching the correct version for oc inside VM particularly the regex which I updated as part of this PR.

Now, what happening is with plain ADB box and without explicitly mentioning any sort of IMAGE_TAG which do in Vagrantfile we ship in CDK.zip, the sccli default downloading the latest openshift docker images.
It happened to downloads the docker images mentioned in this [comment](https://github.com/projectatomic/vagrant-service-manager/issues/357#issuecomment-236536823).

Hope it would make sense to pinned down the tag here. Now, I found that these configurations will only work if these mentioned for all `image_tag`, `image_name` and `docker_registry`.